### ISSUE TITLE
Icon text alternative is not accessible

### ISSUE THREAD

**Issue by garethslinn** [w3c/coga#353]
### Context
To enhance the accessibility of the document for screen readers, particularly focusing on how icons are described. This change is intended as a guide to improve the user experience for visually impaired users.

### Issue
The 'start here' icon contains graphic text that is not accessible to a screen reader.  Due to the method of implementation i.e. css background we can approach this with minimal disruption.  

### Current CSS Implementation

```css
#user_needs section.objective:nth-of-type(1) h3::before {
    content: url(img/StartHere.svg);
}
```

### Suggested enhancement

```css
#user_needs section.objective:nth-of-type(1) h3::before {
    display: inline-block;
    background: url(img/StartHere.svg) no-repeat;
    background-size: contain;
    content: "Start Here: "; 
    color: transparent; 
}
```

### Note
This works well in VoiceOver however it may require some adjustment and rigorous testing on different platforms.

**Comment by lseeman** [comment]
flag for testing to @eric

**Comment by ehind67** [comment]
@ehind67 to test

**Comment by ehind67** [comment]
The icon in this case is really just decorative and there would be minmal value in providing screen reader text.  The other icons in the Summary do not have focus or screenreader text.

<img width="993" height="674" alt="Image" src="https://github.com/user-attachments/assets/3ba3c652-7858-49fa-999d-ba552c04d3b3" />

Propose closing as working as intended.

**Comment by ehind67** [comment]
Reassigned to @lseeman to OK before closing.
@julierawe This summary may not exist in the new Content Usable anyways?
